### ISSUE TITLE
Is this project still active ?

### ISSUE THREAD

**Issue by DittelHome** [rdpoor/rigol-grab#14]
5 years without any changes ...

**Comment by rdpoor** [comment]
Yes - I just added support for the DHO924.  Tested under windows, soliciting comments for MacOS and Linux.

> 5 years without any changes 

What changes would you like to see?
